inferentia/README.md

```diff
@@ -202,4 +227,22 @@ Note:
 1. The `config.pbtxt` and `model.py` should be treated as
 starting point. The users can customize these files as per
 their need.
-2. Triton Inferentia is currently tested with a **single** model. 
\ No newline at end of file
+2. Triton Inferentia is currently tested with a **single** model. 
+
+## Testing Inferentia Setup for Accuracy
+The [qa folder](https://github.com/triton-inference-server/python_backend/tree/main/inferentia/qa)
+contains the necessary files to set up testing with a simple add_sub model. The test
+requires an instance with more than 8 inferentia cores to run, eg:`inf1.6xlarge`.
+start the test, run 
+```
+source <triton path>/python_backend/inferentia/qa/setup_test_enviroment_and_test.sh
+``` 
+where `<triton path>` is usually `/home/ubuntu`/.
+This script will pull the corresponding [server](https://github.com/triton-inference-server/server)
+that contains the tests for inferentia. It will then build the most recent 
+Triton Server and Triton SDK. 

```

Comment:
I do understand we are using perf_analyzer for output validation for some models. I believe we must also enable L0_infer in for inferentia models as it would test along all the range of datatypes and must take lesser time than perf_analyzer tools.

Reply:
since it is going to take more work to set up each model with gen script, would you be ok with adding this to a separate PR?